vizro-ai/docs/pages/user-guides/genAI-use-case.md

```diff
@@ -0,0 +1,103 @@
+# Vizro-AI use case
+
+This guide provides instructions on incorporating a chart generated by VizroAI into your [vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
+
+First step is to create a chart illustrating the population development of each continent over time. To gain deeper insights and access the underlying code responsible for generating the chart, include explain=True in the plot() method. Let's execute the provided code and examine the outcome.
+
+!!! example "Vizro-AI chart"
+    === "Code for the cell"
+        ```py
+        import vizro_ai
+        from vizro_ai import VizroAI
+        import plotly.express as px
+        from dotenv import load_dotenv
+
+        load_dotenv()
+
+        df = px.data.gapminder()
+        vizro_ai = VizroAI(model_name="gpt-4-0613")
+
+        vizro_ai.plot(df,
+                      """Plot a bubble chart to show the changes in life expectancy
+                       and GDP per capita for each country over time.
+                       Color the bubbles by continent.
+                       Add animation on yearly basis, and do not use facets.
+                       Put the legend on top""")
+        ```
+    === "Result"
+        [![VizroAIChart]][VizroAIChart]
+
+    [VizroAIChart]: ../../assets/user_guides/vizro-ai-chart.gif
+
+
+Once you are satisfied with the chart, we can proceed with the next step, and that is incorporating the chart into [vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
+
+In the explanation section, you'll find a code snippet used to generate the chart.
+
+```py
+@capture('graph')
+def custom_chart(data_frame=pd.DataFrame()):
+    # Create a bubble chart
+    fig = px.scatter(data_frame, x='gdpPercap', y='lifeExp', animation_frame='year', animation_group='country',
+                     size='pop', color='continent', hover_name='country', log_x=True, size_max=60,
+                     range_y=[20,90],
+                     labels={'gdpPercap':'GDP per Capita', 'lifeExp':'Life Expectancy', 'year':'Year', 'continent':'Continent', 'pop':'Population'},
+                     title='Life Expectancy v. Per Capita GDP, 1952-2007',
+                     height=400)
+    # Put the legend on top
+    fig.update_layout(legend=dict(y=1, yanchor='top', x=0.5, xanchor='center'))
+    # Return the plot
+    return fig
+```
+
+Simply copy and paste the custom chart code as it can be used directly in the [vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
```

Comment:
```suggestion
Simply copy and paste the custom chart code as it can be used directly in the [Vizro](https://github.com/mckinsey/vizro/tree/main/vizro-core) dashboard.
```